src/main/java/teammates/common/util/Config.java

```diff
@@ -78,6 +78,13 @@ public String getAppVersion() {
         return SystemProperty.applicationVersion.get().split("\\.")[0].replace("-", ".");
     }
 
+    /**
+     * @return true if the system is running at development environment
+     */
+    public boolean isDevEnvironment() {
```

Comment:
Yes it can, but I'm following how the two methods above this (the ones that involve `SystemProperty`) are non-static.